isis/src/osirisrex/objs/OsirisRexOcamsCamera/unitTest.cpp

```diff
@@ -59,33 +59,33 @@ int main(void) {
   try {
 
     cout << "\nTesting PolyCam (backwards compatibility)..." << endl;
-    Cube polyCamCube("$osirisrex/testData/2019-01-13T23_36_05.000_PCAM_L2b_V001.cub", "r");
+    Cube polyCamCube("$ISISTESTDATA/../testDataLocal/isis/src/osirisrex/unitTestData/2019-01-13T23_36_05.000_PCAM_L2b_V001.cub", "r");
```

Comment:
is testDataLocal on all machines? Seems new to me and I think it might have to be added to the Jenkins Mac

Reply:
Fixed anyway, they needed to be in $ISISTESTDATA